src/main/java/n/N.java

```diff
@@ -0,0 +1,215 @@
+            + "|___| \\____| .chatbot :)";
+    static Task[] taskList = new Task[MAX_LIST_ITEMS];
+    static int taskCount = 0;
+
+    public static void printLine() {
+        System.out.println("    ____________________________________________________________\n");
+    }
+
+    public static void printMessage(String message) {
+        printLine();
+        System.out.println("    " +message);
+        printLine();
+    }
+
+    public static void printTaskList() {
+        if (taskCount == 0) {
+            printMessage("no tasks added, wake up pleasee!");
+        } else {
+            printLine();
+            System.out.println("    Here are the tasks in your list:");
+            for(int i = 0; i < taskCount; i++) {
+                System.out.println("    " + taskList[i]);
+            }
+            printLine();
+        }
+    }
+
+    public static void changeTaskStatus(int taskIndex, boolean newStatus) {
+        String outputMessage;
+        //check to ensure that task to be marked/unmarked exists in the list
+        if (taskIndex < taskCount) {
+            //when no change in status is required
+            if (taskList[taskIndex].isDone() == newStatus) {
+                //generate output message based on current task status
+                outputMessage = (newStatus) ?
+                        //output message if task has already been marked done
+                        "Task " +taskList[taskIndex].getIndex()+ " is already completed!" :
+                        //output message if task has not been marked done
+                        "Task " +taskList[taskIndex].getIndex()+ " is already unmarked, complete other tasks!";
+            } else { //handle a change in task status
+                taskList[taskIndex].setDone(newStatus);
+                //generate output message based on new task status
+                outputMessage = (newStatus) ?
+                        //output message when task is marked done
+                        "Task " +(taskIndex + 1)+ " marked done, yay! :)" :
+                        //output message when task is unmarked
+                        "Okay, task " +(taskIndex + 1)+ " unmarked, let's complete it soon ~.o.~";
+            }
+        } else {
+            outputMessage = "Task not found :p";
+        }
+        printMessage(outputMessage);
+    }
+
+    public static Type filterTask(String taskDescription) throws NoTaskTypeException {
+        Type taskType;
+        switch (taskDescription.split(" ")[0]) {
+            case "event":
+                taskType = Type.Event;
+                break;
+            case "deadline":
+                taskType = Type.Deadline;
+                break;
+            case "todo":
+                taskType = Type.ToDo;
+                break;
+            default:
+                throw new NoTaskTypeException();
+        };
+        return taskType;
+    }
+
+    public static void addToTaskList(Type taskType, String message)
```

Comment:
this function is too long (>30 LOC) , maybe consider separating some of the code snippets into different functions.
https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods